src/main/clojure/aerospike_clj/protocols.clj

```diff
@@ -0,0 +1,170 @@
+(ns aerospike-clj.protocols
+  (:refer-clojure :exclude [update]))
+
+(defprotocol AerospikeReadOps
+  "Read operations of a single Aerospike record."
+  (get-single
+    [this index set-name]
+    [this index set-name conf]
+    [this index set-name conf bin-names]
+    "Returns a single record: `(transcoder AerospikeRecord)`. The default transcoder is `identity`.
+    Pass a `:policy` in `conf` to use a non-default `ReadPolicy`.")
+
+  (get-single-no-meta
```

Comment:
We can do that for many of the methods in the protocols, as most of them merely supply a specific policy and/or transcoder. Having said that, they were already part of the API since day 1 and are handy, so I'm not in favor of removing them.